cmd/integration_link.go

```diff
@@ -447,6 +494,18 @@ func interactiveCreate() (scalingo.SCMRepoLinkCreateParams, error) {
 		hoursBeforeDestroyOnStale := uint(hoursBeforeDestroyOnStale64)
 		params.HoursBeforeDeleteStale = &hoursBeforeDestroyOnStale
 	}
+
+	forksAllowed := false
+	err = survey.AskOne(&survey.Confirm{
+		Message: "Allow review apps to be created from forks:",
+		Help:    "Only allow automatic review apps deployments from forks if you trust the owners of those forks, as this could lead to security issues",
+		Default: forksAllowed,
+	}, &forksAllowed, nil)
```

Comment:
I like the use of `io.Warning` rather than the help mesage to make sure the warning is always seen. We could do the same here.

Also we could then use the same message defined in the constant `reviewAppsFromForksSecurityWarning` (`warningSecurityMessageForAutomaticReviewAppsFromForks`)

Reply:
Nice idea, it has been done

Reply:
> it has been done

I don't see the change here. Am I missing something?

Reply:
Oh I got messed up, way a bit